### PR TITLE
Vars and fields renaming 

### PR DESCRIPTION
**Pull request by altafan**
This replaces all occurrences of legacy "asp" to "server", "pool" to "round", "congestion tree" to "vtxo tree", and "payment" to "request" at any level.

This also consolidates naming for pubkey/prvey vars and struct fields: vars always lower-case `pubkey` or `Pubkey`, struct fields always camel-case `PubKey`.

NOTE: this contains breaking changes for server and client at db level. 

Please @louisinger  @tiero review this.

### CODE REVIEW

**Review by louisinger — APPROVED** [review]

**Comment by tiero** [comment]
what about congestionTree in common? As we are not using degrading timelocks (yet)

File: api-spec/protobuf/ark/v1/service.proto [hunk 1]
```diff
@@ -143,13 +145,13 @@ message RegisterInputsForNextRoundRequest {
 }
 message RegisterInputsForNextRoundResponse {
   // Mocks wabisabi's blinded credentials.
```
Comment (tiero) [inline api-spec/protobuf/ark/v1/service.proto]: remove this comment

File: api-spec/protobuf/ark/v1/service.proto [hunk 2]
```diff
@@ -143,13 +145,13 @@ message RegisterInputsForNextRoundRequest {
...
-  string id = 1;
+  string request_id = 1;
 }
 
 message RegisterOutputsForNextRoundRequest {
   // Mocks wabisabi's blinded credentials.
```
Comment (tiero) [inline api-spec/protobuf/ark/v1/service.proto]: remove this comment

File: api-spec/protobuf/ark/v1/service.proto [hunk 3]
```diff
@@ -171,7 +173,7 @@ message SubmitTreeSignaturesResponse {}
 message SubmitSignedForfeitTxsRequest {
   // Forfeit txs signed by the user. 
   repeated string signed_forfeit_txs = 1;
-  // If payment has boarding input, the user must sign the associated inputs.
+  // The user has to sign also the round tx if he's onboarding.
```
Comment (tiero) [inline api-spec/protobuf/ark/v1/service.proto]:
```suggestion
  // The user has to sign also the round tx if it register a boarding UTXO
```

**Pull request merged**